src/forms/NewStudyForm.js

```diff
@@ -45,6 +56,33 @@ const NewStudyForm = ({
       href: 'logistics',
     },
   ];
+  const missingValueMessage = values => {
+    var fields = [];
+    trackedStudyFields.map(field => {
+      if (!values[field] || values[field].length === 0 || values[field] === 0) {
+        fields.push(fieldLabel[field]);
```

Comment:
is there a way to map what field goes to what step so we can link them to the step the field is in?